paimon-core/src/main/java/org/apache/paimon/table/ExpireChangelogImpl.java

```diff
@@ -79,6 +87,12 @@ public ExpireSnapshots config(ExpireConfig expireConfig) {
 
     @Override
     public int expire() {
+        TableSchema latestTableSchema = this.schemaManager.latest().get();
```

Comment:
This will bring additional IO access. Can you introduce a new option to describe whether a refresh is needed?